src/www/ui/user-edit.php

```diff
@@ -55,6 +55,14 @@ protected function handle(Request $request)
     $SessionUserRec = $this->GetUserRec($user_pk);
     $SessionIsAdmin = $this->IsSessionAdmin($SessionUserRec);
 
+    if (!($SessionIsAdmin or
```

Comment:
It won't be defined for $SessionIsAdmin==true due to [lazy evaluation](https://en.wikipedia.org/wiki/Lazy_evaluation).
Equivalently, `(!(true or empty($a=2)) or 2==$a)` will be false (and throw a notice) while `(!(true | empty($a=2)) or 2==$a)` is true.
Moreover, [empty()](http://php.net/manual/en/function.empty.php) supporting expressions, rather than only variables, is a new feature of php 5.5.0